docs/publishing.md

```diff
@@ -0,0 +1,105 @@
+# How to publish Jenkins agent deb PPA
+
+This document will talk about how to bootstrap a debian package repo, what are its components, 
+what they do and how to build, test, sign and publish a debian package. 
+https://github.com/canonical/jenkins-agent-deb will be used as an example.
+
+This document is based on https://wiki.debian.org/Packaging/Intro and goes more into details about
+the packaging process. It also tries to explain some hidden convention in the packaging process and
+put the packaging into the context of the Platform Engineering team.
+
+### Install necessary packages
+
+```
+sudo apt install build-essential devscripts debhelper
+```
+
+### Bootstrap the package repo
+
+1. debian/changelog
```

Comment:
For my understanding, debian/changelog is a file? How does the `dch` command below create the debian/changelog? 

I would suggest adding a verb to this step, e.g., "Create the debian/changelog", or whatever action makes the most sense for this step